src/consent.js

```diff
@@ -86,32 +85,14 @@ export function getConsentPolicyInfo(element, policyId) {
  */
 export function shouldBlockOnConsentByMeta(element) {
   const ampdoc = element.getAmpDoc();
-  let content = Services.documentInfoForDoc(ampdoc).metaTags[
-    'amp-consent-blocking'
-  ];
-
+  let content = ampdoc.getMetaByName('amp-consent-blocking');
```

Comment:
Nice! I see a lot of other dedupe opportunities when searching for "meta[name=" in the codebase. Would you mind filing a cleanup issue for this?

Reply:
This? https://github.com/ampproject/amphtml/issues/26652

Reply:
🥇

Reply:
🥇Thanks! much cleaner